kubelet/datadog_checks/kubelet/kubelet.py

```diff
@@ -300,15 +302,16 @@ def perform_kubelet_query(self, url, verbose=True, timeout=10, stream=False):
         """
         Perform and return a GET request against kubelet. Support auth and TLS validation.
         """
-        return requests.get(
-            url,
-            timeout=timeout,
-            verify=self.kubelet_credentials.verify(),
-            cert=self.kubelet_credentials.cert_pair(),
-            headers=self.kubelet_credentials.headers(url),
-            params={'verbose': verbose},
-            stream=stream,
-        )
+        with disable_warnings_ctx(InsecureRequestWarning, disable=not self.kubelet_credentials.verify()):
```

Comment:
Could you add a test for this case ? 

Some examples here: https://github.com/k2v/integrations-core/blob/ecd9fa77ece76a50f3a9ce53c8c83224ae343bb1/datadog_checks_base/tests/test_openmetrics.py#L2101-L2117